test/Renci.SshNet.IntegrationTests/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.20
+FROM alpine:latest
```

Comment:
Didn't the original issue show that using latest is a bad idea? It made CI suddenly fail without an obvious indication why. Imho it would make more sense to tag it to 3.21.1 (even the patch) and use Dependabot to keep it up-to-date.

Reply:
This is the first such issue in ~1.5 years of these integration tests. I don't think we necessarily need to make a process out of it, but if it's simple enough to do then sure

Reply:
well it was simple enough